bin/aq.py

```diff
@@ -347,11 +347,11 @@ def get_default_opts(auth_option, conf_file=None):
         default_aqhost = defaultOpts.get('aqhost') or 'nyaqd1'
         default_aqservice = 'cdb'
     else:
-        default_aqhost = socket.gethostname()
+        default_aqhost =  socket.gethostname()
         default_aqservice = get_username()
 
-    host = globalOptions.get('aqhost') or os.environ.get('AQHOST', None) or \
-        default_aqhost
+    host = globalOptions.get('aqhost') or defaultOpts.get('aqhost') or \
+           os.environ.get('AQHOST', None) or default_aqhost
```

Comment:
Yes you are right - env variables should override the default. Therefore they should go first :) I do not remember if i told u an order - but it should be:
```
host = globalOptions.get('aqhost') or os.environ.get('AQHOST', None) or \
 defaultOpts.get('aqhost') or default_aqhost
```